edwardsTicApp/Db/edwardsTicGauge.db

```diff
@@ -0,0 +1,77 @@
+# edwardsTicGauge.template
+# Template database for for the Edwards TIC Turbo Instrument Controller
+# Gauge
+# Pete Owens 15/2/2017
+#
+# substitutions:
+# device  - device name
+# port    - asyn port
+# gauge   - gauge num (1-3)
+#
+################################################################################
+# P - Pressure
+#
+record(ai, "$(P)GAUGE$(GAUGE):P")
```

Comment:
Our [standard signal names convention](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming#standard-signal-names) suggests using `PRESSURE` - I think `P` is a bit too cryptic here. If `PRESSURE` is too long and we're hitting name length limits can we use `PRES`